src/style.css

```diff
@@ -2,0 +1,69 @@
+body {
+  margin: 0 8px;
+  font-family: Avenir, Arial, sans-serif;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: 700;
+  font-style: normal;
+}
+
+.input {
+  margin-bottom: 20px;
```

Comment:
According to the checklist, it's recommended to be consistent with your margins (Add only top or bottom, but not both) to avoid potential margin collapse. Here you are adding margin to the bottom of the input, which is not consistent with the checklist guidelines.